.github/workflows/dependency-graph.yml

```diff
@@ -0,0 +1,20 @@
+# .github/workflows/dependency-graph.yml
+name: Update Dependency Graph
+
+on:
+  push:
+    branches:
+      - main # default branch of the project
+  workflow_dispatch:
```

Comment:
Is this line needed?

Reply:
with workflow_dispatch you should be able to run it manually. don't know. I thought it would be useful.

Reply:
I removed it.